runtime/cpuset/cpuset_builder.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package cpuset
+
+import (
+	"fmt"
+)
+
+// Builder will allow building of cpuset fields
+// http://man7.org/linux/man-pages/man7/cpuset.7.html
```

Comment:
I think it makes more sense to link to the cpuset docs near the definition of the CPUSet structure, rather than this builder. Or maybe in some introductory text in the file header in which we explain what cpusets are and how we use them.

Reply:
Good call out. Will go ahead and do that.